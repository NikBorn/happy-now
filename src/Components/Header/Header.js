import React from 'react';
import SignIn from '../SignIn/SignIn';
import { connect } from 'react-redux';
import PropTypes from 'prop-types';


const Header = (props) => {

  return (
    <div className='header-section'>
      <h1>HappyNow?</h1>
      <h4>Find current and upcoming happy hour specials near you</h4>
      <SignIn />
    </div>
  );
};


export default Header;