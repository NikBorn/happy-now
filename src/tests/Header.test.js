import React from 'react';
import { shallow, mount, configure } from 'enzyme';
import Header from '../Components/Header/Header.js';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() })

describe('Header', () => {

  let wrapper = shallow(<Header/>)

  it('should create an instance of a card', () => {
    expect(wrapper).toMatchSnapshot();
  });

})