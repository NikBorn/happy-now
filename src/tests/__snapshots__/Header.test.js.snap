// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should create an instance of a card 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Header />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": null,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1>
          HappyNow?
</h1>,
        <h4>
          Find current and upcoming happy hour specials near you
</h4>,
        <Connect(SignIn) />,
        <button
          className="show-favorites-button"
>
          Favorites
</button>,
      ],
      "className": "header-section",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": null,
        "nodeType": "host",
        "props": Object {
          "children": "HappyNow?",
        },
        "ref": null,
        "rendered": "HappyNow?",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": null,
        "nodeType": "host",
        "props": Object {
          "children": "Find current and upcoming happy hour specials near you",
        },
        "ref": null,
        "rendered": "Find current and upcoming happy hour specials near you",
        "type": "h4",
      },
      Object {
        "instance": null,
        "key": null,
        "nodeType": "class",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": null,
        "nodeType": "host",
        "props": Object {
          "children": "Favorites",
          "className": "show-favorites-button",
        },
        "ref": null,
        "rendered": "Favorites",
        "type": "button",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": null,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1>
            HappyNow?
</h1>,
          <h4>
            Find current and upcoming happy hour specials near you
</h4>,
          <Connect(SignIn) />,
          <button
            className="show-favorites-button"
>
            Favorites
</button>,
        ],
        "className": "header-section",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": null,
          "nodeType": "host",
          "props": Object {
            "children": "HappyNow?",
          },
          "ref": null,
          "rendered": "HappyNow?",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": null,
          "nodeType": "host",
          "props": Object {
            "children": "Find current and upcoming happy hour specials near you",
          },
          "ref": null,
          "rendered": "Find current and upcoming happy hour specials near you",
          "type": "h4",
        },
        Object {
          "instance": null,
          "key": null,
          "nodeType": "class",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": null,
          "nodeType": "host",
          "props": Object {
            "children": "Favorites",
            "className": "show-favorites-button",
          },
          "ref": null,
          "rendered": "Favorites",
          "type": "button",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
